code/global.dm

```diff
@@ -1,5 +1,7 @@
 #define DNA_SE_LENGTH 55
 
+#define vox_shaped "Vox","Skeletal Vox"
```

Comment:
This define needs to be capitalized fully. All defines are capitalized fully excepting macros.